frontend/app/mixins/model-truncated-details.js

```diff
@@ -1,23 +1,25 @@
 import Ember from 'ember';
 
 export default Ember.Mixin.create({
-    truncatedBio: function(key, value) {
-      // setter
-      if (arguments.length > 1) {
+    truncatedBio: Ember.computed('bio', {
```

Comment:
We can merge this without that, right?  I'd say LGTM, this looks pretty damn solid.  It does pass the "poke test", right?  You poked at it a bit and checked that it wasn't exploded in important places (profiles, feeds, libraries)?